packages/fhir-care-team/src/components/CreateEditCareTeam/utils.tsx

```diff
@@ -39,30 +41,32 @@ export const submitForm = async (
 ): Promise<void> => {
   const { initialCareTeam } = values;
   const { meta, text, participant, ...nonMetaFields } = initialCareTeam ?? {};
-  const carriedOverParticipants = (participant ?? []).filter(
-    (participant) =>
-      !participant.member?.reference?.startsWith(organizationResourceType) ||
-      !participant.member.reference.startsWith(practitionerResourceType)
-  );
-  const careTeamId = uuid ? uuid : v4();
 
-  const practitionerParticipantPayload = values[practitionerParticipants].map((id) => {
-    return {
+  const carriedOverParticipantsById = keyBy(participant, (res) => res.member?.reference) as Record<
```

Comment:
We should probably have a conversation about retiring lodash. I feel like js/ts is mature enough to abandon lodash

Reply:
Feel that abandon is a strong word for this case, but yeah we can definitely have that discussion.

Reply:
https://github.com/opensrp/web/issues/1177